### ISSUE TITLE
[FUG] Movies added from RSS-Lists (IMDB) are not shown on the movie page or in search

### ISSUE THREAD

**Issue by Grimeton** [Radarr/Radarr#1439]
Hi,

when movies are added via rss synced lists like this one:

![grafik](https://cloud.githubusercontent.com/assets/2773919/25379226/a248095c-29ac-11e7-9618-0122c67a8371.png)

then the movies are neither listed on the movies page nor are they shown in a search.

So on would figure the movie hasn't been added and when going to the "add movies" page and search for the movie the system offers you to add the movie to the database. But once you click on the "+" button, you get the message that the movie has already been added:

![grafik](https://cloud.githubusercontent.com/assets/2773919/25379318/ecbe0b3a-29ac-11e7-8d92-cc370275a9f0.png)

So it looks like the movie is known to Radarr "internally" but not shown anywhere.

Am I missing something or is this a (F)*cking B(UG) ?

KR,

G.

**Comment by Grimeton** [comment]
Hi again,

of course I forgot to add the version number. While checking for it I saw that there's an update available. Installed the update and now the movies are shown.

Not sure if this is caused because it has been fixed in the update or because I restarted Radarr.

If the former, please close the bug.

Current version now: 0.2.0.654 - Apr 18 2017 

Sorry.

KR,

G.

**Comment by galli-leo** [comment]
@Grimeton Probably caused by the restart. Did you try refreshing the page and then checking again if the movies are there?

**Comment by Grimeton** [comment]
Hi,

the movies in the RSS list have been added there days ago. And the page was reloaded several times since then.

The thing is: after the update all the movies from the rss feed are actually shown. Before the update none of them were shown. I had the suspicion that it was caused by pressing the "+" button on the "add movies" page but that's not true.

I also tested. Added new movies to the list. Refreshed the list (Net Import Sync) and then reloaded the page and the movies are there. 

Right now I can see all the movies that are on any RSS feed I use. So it either magically fixed itself or something changed after the update.

Nevermind. I keep an eye on it and if it happens again, I'll reopen a bug.

Cu
